samples/MegaDrone/src/main/cpp/native-lib.cpp

```diff
@@ -48,7 +48,8 @@ JNIEXPORT jlong JNICALL
 Java_com_example_oboe_megadrone_MainActivity_startEngine(JNIEnv *env, jobject /*unused*/,
                                                          jintArray jCpuIds) {
     std::vector<int> cpuIds = convertJavaArrayToVector(env, jCpuIds);
-    MegaDroneEngine  *engine = new MegaDroneEngine(cpuIds);
+    LOGD("cpu ids size: %lu", cpuIds.size());
```

Comment:
This is failing in Travis. Format mismatch long vs int

Reply:
Out of interest, how did you know this was failing - I checked Travis and it showed the build as passing, however, checking the build log it showed a compilation error. I've filed a bug for this. 

Regardless, good spot. This is caused by `size_type` being 64 bits or 32 bits depending on target architecture - I only tested on 64 bits. @atneya's solution to cast to an int seems to be the best solution.